Form-Controls/index.html

```diff
@@ -8,20 +9,52 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <form>
+      <label for="customer-name">Your Name: </label>
+      <input type="text" name="customer-name" id="customer-name" minlength="2" required>
+
+      <br><br>
+      <label for="email">Your Email: </label>
+      <input type="email" name="email" id="email" required>
+
+      <br><br>
+      <!--Color options-->
+      <label for="color-select">Choose Your Preferred Color: </label>
+      <select id="color-select" name="color">
```

Comment:
Nice - this does the job. Can you think of an input type that could give the user more information here? Are there ways to show them what type of red they are choosing?

Reply:
The input of type color?

Reply:
Right - and if you combine it with datalist you can make a swatch https://demo.agektmr.com/datalist/